docs/constants.md

```diff
@@ -0,0 +1,25 @@
+# Parthenon Built-in Physical Constants
+
+PhysicalConstants is a class that provides numerical values of physical
+constants in common unit systems (SI and CGS). This is provided in the hopes
+that parthenon-derived applications can conveniently share exactly matching
+values for these constants, to avoid bugs that can be associated with
+disagreement and the effort required to synchronize a consistent move to newer
+values of physical constants.
+
+Internal values are hardcoded in SI units, and specific unit systems are
```

Comment:
SI as the base unit system isn't unreasonable. I'm curious why you chose that over cgs though.

Reply:
SI is just the units they're provided in at https://physics.nist.gov/cuu/Constants/

Reply:
Ah okay. Yeah that makes sense.